### ISSUE TITLE
stepwise learning rate scheduler

### ISSUE THREAD

**Issue by Selimonder** [PyTorchLightning/pytorch-lightning#4466]
Hello,

I am trying to manage LR scheduling with my training with pL. Main methods are relying on epoch-wise updates on LR, is there a way to reduce this to step-wise?

 - OS: [Linux]
 - Packaging [pip]
 - Version [1.0.4]

**Comment by ayasyrev** [comment]
https://pytorch-lightning.readthedocs.io/en/latest/optimizers.html?highlight=scheduller
for example:
def configure_optimizers(self):
   optimizer = Adam(...)
   scheduler = {'scheduler': OneCycleLR(optimizer, ...), 
                        'interval': 'step'}
    return [optimizer], [scheduller]

**Comment by s-rog** [comment]
I'll close this for now, feel free to re-open if you have more questions.

Also check out our forums: https://forums.pytorchlightning.ai/
